### PR TITLE
net-misc/youtube-viewer: Bump to new 3.2.0 release

### PR DESCRIPTION
**Pull request by zpuskas**
Package-Manager: portage-2.2.27

### CODE REVIEW

**Comment by zpuskas** [comment]
@jlec 
@idella 
@gentoo/proxy-maint

File: net-misc/youtube-viewer/youtube-viewer-3.2.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit gnome2-utils eutils perl-module vcs-snapshot
```
Comment (jlec) [inline net-misc/youtube-viewer/youtube-viewer-3.2.0.ebuild]: I like to have this sorted. But this isn't a must. Jut more convenient to read.

File: net-misc/youtube-viewer/youtube-viewer-3.2.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+pkg_preinst() {
+	use gtk && gnome2_icon_savelist
+	perl_set_version
+}
+
+pkg_postinst() {
+	use gtk && gnome2_icon_cache_update
+	elog
+	elog "optional dependencies:"
```
Comment (jlec) [inline net-misc/youtube-viewer/youtube-viewer-3.2.0.ebuild]: You could use ```optfeature()``` from ```eutils.eclass``` here.

File: net-misc/youtube-viewer/youtube-viewer-3.2.0.ebuild (hunk 1 above)
Comment (idella) [inline net-misc/youtube-viewer/youtube-viewer-3.2.0.ebuild]: you mean use alpha order?

Comment (jlec) [inline net-misc/youtube-viewer/youtube-viewer-3.2.0.ebuild]: yes.

**Pull request merged**